### PR TITLE
Dashboard tutorial v8

### PR DESCRIPTION
**Pull request by jmayntzhusen**
Updated the dashboard tutorial to Umbraco 8.

### CODE REVIEW

**Comment by jmayntzhusen** [comment]
@marcemarc I had to change some of the examples due to changes to differences in the getPagedUserLog method. Would you mind having a look at this? https://github.com/umbraco/UmbracoDocs/pull/1967/files#diff-61c19734b9eb675a2fab135d98915ce2R364-R394

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 1]
```diff
@@ -165,7 +181,7 @@ We register this AngularJS controller to the Umbraco Angular module:
 ```js
 angular.module("umbraco").controller("CustomWelcomeDashboardController", function ($scope) {
     var vm = this;
-    alert('hello world');
+    alert("hello world");
```
Comment (sofietoft) [inline Tutorials/Creating-a-Custom-Dashboard/index.md]: In line 195:
Sentence is a bit weird.
Should it be something like this instead perhaps:
"The use of vm (short for view model) is to enable ..."

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 2]
```diff
@@ -198,7 +224,7 @@ If all is setup fine we should now receive the 'Hello world' alert every time th
 
 ### Going further - Umbraco Angular Services and Directives
 
-Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/apidocs/v7/ui/#/api
+Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/apidocs/v8/ui/#/api
```
Comment (sofietoft) [inline Tutorials/Creating-a-Custom-Dashboard/index.md]: Line 229:
"Umbraco is a place where everybody knows your name..." - a bit creepy if you ask me.

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 3]
```diff
@@ -222,17 +254,17 @@ var user = userService.getCurrentUser().then(function (user) {
 Finally we can now update our view to incorporate the current user's name in our Welcome Message:
```
Comment (sofietoft) [inline Tutorials/Creating-a-Custom-Dashboard/index.md]: Above this, there's a **Tip**
Shouldn't this be in note form instead? 🤔

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 4]
```diff
@@ -222,17 +254,17 @@ var user = userService.getCurrentUser().then(function (user) {
...
 
-![Custom Dashboard Welcome Message With Current User's Name](images/welcomemessagepersonalised.jpg)
+![Custom Dashboard Welcome Message With Current User's Name](images/welcomemessagepersonalised-v8.png)
 
 and for reference the full contents of /customwelcomedashboard.controller.js at this stage of the tutorial should look like this:
 
 ```js
 angular.module("umbraco").controller("CustomWelcomeDashboardController", function ($scope,userService) {
     var vm = this;
-    vm.UserName = 'guest';
```
Comment (sofietoft) [inline Tutorials/Creating-a-Custom-Dashboard/index.md]: This code example, is the exact same snippet as the one on lines 204 to 248, isn't it?
Is it necessary to have it two times in a row? 🤔

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 5]
```diff
@@ -34,6 +32,7 @@ So all the steps we will go through:
 - You can do anything...
 
 ### Prerequisites
+
```
[the remark this answers is not in the document]
Comment (jmayntzhusen) [reply Tutorials/Creating-a-Custom-Dashboard/index.md]: I would say it's still fine. A lot of the property editor things are the same or very similar in Umbraco 8, and that way we don't have to remember to go back and update this later..

File: Tutorials/Creating-a-Custom-Dashboard/index.md (hunk 2 above)
Comment (jmayntzhusen) [reply Tutorials/Creating-a-Custom-Dashboard/index.md]: Haha that's fair, it's from the original v7 tutorial, I didn't change it, but will remove that part for now 😁

File: Tutorials/Creating-a-Custom-Dashboard/index.md (hunk 3 above)
Comment (jmayntzhusen) [reply Tutorials/Creating-a-Custom-Dashboard/index.md]: Good call, will add it as one of our custom tips instead: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

File: Tutorials/Creating-a-Custom-Dashboard/index.md (hunk 2 above)
Comment (marcemarc) [reply Tutorials/Creating-a-Custom-Dashboard/index.md]: https://www.youtube.com/watch?v=h-mi0r0LpXo

**Comment by marcemarc** [comment]
@jmayntzhusen The whole bit that shows what the logged in user has been doing from getPagedUserLog... has the key learning objectives of

1) You can inject angular resources and services into your controller
2) Umbraco has a whole bunch of useful resources and services that you can use that are documented here...

so it was quite arbitrary to use this example, but seemed to fit with the intent of the dashboard...

the original AuditUserLog, didn't really store the information how I'd have expected it! to display the list... so this bit has always felt a bit clunky, but still hopefully achieving the key learning objectives...

in moving to V8 and variants, then yes we need to reflect this change, in this dashboard.. as it's now part of the core and we want an example that is useful... so be good to show in the list if it's a variant that has been saved etc and if the link to the page, takes you to that particular variant, that would be good... so it's likely that using the getPageUserLog and writing out the list will still be a bit klunky, but as long as that doesn't detract from the key learning objectives, and doesn't provide something that isn't functional as the end product of the dashboard tutorial... if variants etc are 'in play'

if that makes sense...

**Comment by jmayntzhusen** [comment]
Thanks for the feedback @marcemarc 🙂

I have updated the example to output the variance change if there was one:
CreateNewBlogPost-v8

**Comment by jmayntzhusen** [comment]
Something weird is going on with my merge conflict fixes and my rebase.. When comparing the fork branch to master it is fine but in this PR it has a ton of additional changes. Will close this PR and resubmit.

**Pull request closed without merging**